docker-compose.yml

```diff
@@ -18,6 +18,9 @@ services:
       - mojaloop-net
     depends_on:
       - mysql
+      - redis
+    extra_hosts:
```

Comment:
Why do we need `extra_hosts`?

Reply:
Don't believe we need it. Just mindless copying. 
Let me know if I set up the compose files properly like the pisp repo.

Reply:
Cool. Yeah it looks fine to me apart from that section. We'll know if the config is wrong because the tests will break.